### DISCUSSION TITLE
Size of the bundle ??

### DISCUSSION THREAD

**Discussion by Vince-ALIEN** [tjtanjin/react-chatbotify#76] [Q&A]
WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets:
  870.js (302 KiB)

is there a way to decrease of this js ? I use webpack...?

**Comment by tjtanjin** [comment]
You can start by identifying what's causing the large size - it's hard to tell what's going on with what you provided
